WTWSMS/events/religious_events.txt

```diff
@@ -191,6 +199,7 @@ character_event = {
 		OR = {
 			religion = catholic
 			religion = fraticelli
+			religion = nicene
```

Comment:
When he barely ruled his own houseblock, he isn't in any measure to send these requests.